src/kivy_garden/zbarcam/zbarcam.py

```diff
@@ -132,6 +132,7 @@ class ZBarCam(AnchorLayout):
     Widget that use the Camera and zbar to detect qrcode.
     When found, the `codes` will be updated.
     """
+    index = 0
```

Comment:
Shouldn't we make it a `NumericProperty` so we can potentially act on event?
Also we may want to call it something more meaningful so it's more clear what it's doing.
Maybe `camera_index`?